app/app/urls.py

```diff
@@ -184,6 +184,7 @@
 
     # modals
     re_path(r'^modal/get_quickstart_video?', dashboard.views.get_quickstart_video, name='get_quickstart_video'),
+    re_path(r'^modal/extend_issue_deadline?', dashboard.views.extend_issue_deadline, name='extend_issue_deadline'),
```

Comment:
If we're using regex with `?`, you'll want to throw a `/` between `e` and `?`.  The route above it is actually wrong - That means `/modal/get_quickstart_vide` hits the same endpoint as `modal/get_quickstart_video` and `modal/get_quickstart_video/`